api/src/main/java/net/akami/yggdrasil/api/game/events/CancelledEventsListener.java

```diff
@@ -65,9 +64,9 @@ public void onDropItem(DropItemEvent event) {
 
     @Listener(order = Order.LAST)
     public void onItemSpawned(SpawnEntityEvent event) {
-        event.
-                filterEntities(entity -> entity.getType() != EntityTypes.ITEM)
-                .forEach(Entity::remove);
+        /*event.getEntities().stream()
+                .filter(entity -> entity.getType() == EntityTypes.ITEM && !entity.get(YggdrasilKeys.PERSISTENT).orElse(false))

```

Comment:
Whoops, I've forgotten to re-enable this part of code.

Reply:
Also, I prefer check the PERSISTENT value because you can explicitly say "Hey, remove me pls" and it seems more logical to remove an item explicitly marked as non-persistent. Only items explicitly marked as persistent should be kept. Other cases (no key/false value) should be removed.

Reply:
Oh right. Just because the optional is present doesn't mean it's true, I didn't see that